src/Url.js

```diff
@@ -52,7 +52,7 @@ const Url = React.createClass({
 
   onClick(event) {
     event.preventDefault();
-    ActionCreator.navigateTo(this.props);
+    Store.dispatch(ActionCreator.navigateTo(this.props));
```

Comment:
But maybe it is okay for such a small lib as component-router. For the sake of simplicity and codebase.